docs/server/requirements.md

```diff
@@ -1,10 +1,10 @@
 # Server Requirements
 
-qooxdoo server is a basic component that runs in many different contexts and
-environments, as it has very little dependencies to the underlying runtime. For
-use in command-line tools and programs you will need a corresponding JavaScript
-interpreter like Node.js or Mozilla Rhino. For use in HTML5 Web Workers you will
-need a browser that supports this technology.
+qooxdoo server is a basic component that runs in many different contexts
```

Comment:
`qooxdoo` used to be always lower case. These days, we're starting to capitalize it in some places, as a proper noun. In this case, it's also at the beginning of a sentence. Let's figure out what we're doing with it, and then be consistent.